myuw/templates/handlebars/card/accounts/tuition.html

```diff
@@ -51,37 +51,43 @@ <h4 class="card-badge-label">Account Credit</h4>
                     </li>
                 </ul>
             </div>
-        {{ else }}  <!-- else, there is a balance -->
-            <div class="card-badge-container">
+        {{else}}  <!-- else, there is a balance -->
+            <div class="card-badge-container tuition">
```

Comment:
per our discussion about the .card-badge-container.tuiton syntax, examine whether you really need this class here. my sense is that you don't.